sample-viewmodel/src/main/java/com/example/molecule/viewmodel/MoleculeViewModel.kt

```diff
@@ -20,31 +20,45 @@ import androidx.compose.ui.platform.AndroidUiDispatcher
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import app.cash.molecule.RecompositionMode.ContextClock
-import app.cash.molecule.launchMolecule
+import app.cash.molecule.moleculeFlow
+import kotlin.time.Duration.Companion.seconds
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.SharingStarted
 import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.WhileSubscribed
+import kotlinx.coroutines.flow.onEach
+import kotlinx.coroutines.flow.stateIn
 
-abstract class MoleculeViewModel<Event, Model> : ViewModel() {
+abstract class MoleculeViewModel<Event, Model>(
+  initialState: Model,
+  started: SharingStarted = SharingStarted.WhileSubscribed(5.seconds),
+  presenter: @Composable (seed: Model, events: Flow<Event>) -> Model,
+) : ViewModel() {
   private val scope = CoroutineScope(viewModelScope.coroutineContext + AndroidUiDispatcher.Main)
 
   // Events have a capacity large enough to handle simultaneous UI events, but
   // small enough to surface issues if they get backed up for some reason.
   private val events = MutableSharedFlow<Event>(extraBufferCapacity = 20)
 
+  private var seed: Model = initialState
+
   val models: StateFlow<Model> by lazy(LazyThreadSafetyMode.NONE) {
-    scope.launchMolecule(mode = ContextClock) {
-      models(events)
-    }
+    moleculeFlow(mode = ContextClock) {
+      presenter(seed, events)
+    }.onEach {
+      seed = it
+    }.stateIn(
+      scope = scope,
+      started = started,
+      initialValue = seed,
```

Comment:
The need for this `initialValue` does make the case that having a `started` parameter on `launchMolecule` is maybe not the worst idea in the world. Hooking it up with the existing API surface is uhh *waves hands* possible, for a certain definition of possible, but not one useful to someone who wants to do what `MoleculeViewModel` is doing